test/utilities/fileSystem.ts

```diff
@@ -21,3 +22,29 @@ export function removeFolder(folderPath: string): ChildProcess {
 
   return childProcess;
 }
+
+export async function createCustomObjects(testSetup: TestSetup): Promise<void> {
+  const projectPath = testSetup.projectFolderPath;
+  const tempFolderPath = testSetup.tempFolderPath;
+  const source = path.join(
+    tempFolderPath!,
+    '..',
+    'test',
+    'testData',
+    'CustomSObjects'
+  );
+  const destination = path.join(
+    projectPath!,
+    'force-app',
+    'main',
+    'default',
+    'objects'
+  );
+  fs.copy(source, destination, { recursive: true }, async error => {
+    if (error) {
+      log(`Failed in copying custom objects ${error.message}`);
```

Comment:
Since this was refactored into a utility function, it might be helpful to log what the source and destination values were.  Maybe change line 45 to the following:
```
log(`Failed in copying custom objects ${error.message}`);
log(`source was: '${source}'`);
log(`destination was: '${destination}'`);
```